tests.py

```diff
@@ -0,0 +1,3 @@
+d = {1: "a", 2: "b"}
+for key, value in d.items():
+    print(key)
```

Comment:
What is this?

Reply:
It just test file I play around with python. How is it get in there?